config/interface/output_elements/supply_biomass.yml

```diff
@@ -85,3 +85,18 @@
   dependent_on:
   description:
   output_element_type_name: category_bar
+- under_construction: false
+  unit: PJ
```

Comment:
The gqueries report values in MJ whereas the mekko expects values in PJ. This mismatch results in gigantic values in the model (as can be seen in the chart below). 🤓 

![image](https://user-images.githubusercontent.com/5946498/127491277-28cbe840-67f0-41bb-9856-5bb4ad370926.png)